eng/Packages.Data.props

```diff
@@ -46,7 +47,7 @@
     <PackageReference Update="Microsoft.Extensions.Logging.Abstractions" Version="2.2.0" />
     <PackageReference Update="Microsoft.Extensions.Options" Version="2.2.0" />
     <PackageReference Update="Microsoft.Extensions.PlatformAbstractions" Version="1.1.0" />
-    <PackageReference Update="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="4.5.1" />
+    <PackageReference Update="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="[3.14.2, 4.5.1]" />
```

Comment:
This was needed to resolve conflicts between the Event Hubs management library and the Track One Event Hubs library, by way of dependencies.   Happy to consider another approach if someone has one; this was the leave invasive change that I was able to make.